### PR TITLE
Initializing apt sources and apt repo signatures

### LINKED ISSUE #3: Initialize apt sources
Need to setup the update maker to initialize the local apt `source.list` file as python3-apt will leave that file blank on first time setup.

Probably can just copy the context of the real root apt `source.list` file into the local one.

Also need to append `deb [trusted=yes] https://debian.oresat.org/packages ./` so the update maker has access to OreSat Debain apt repo.

### PR DESCRIPTION
**Pull request by mairacanal**
Closes #3 

Initializing apt sources, adding the OreSat Debian apt repo to sources.list, and copying the apt repo signatures. I couldn't find any GitHub practices manual or a set of specific coding guidelines, so I tried to replicate @ryanpdx practices. 

If you have any kind of comments or suggestions, please write below.

### CODE REVIEW

File: update_maker/update_maker.py [hunk 1]
```diff
@@ -45,12 +50,24 @@ def __init__(self, board: str):
...
+        # copying the context of the real root apt source.list file into the local one
+        copyfile(SYSTEM_APT_SOURCES_FILE, OLU_APT_SOURCES_FILE)    
+
+        # adding OreSat Debian apt repo
+        with open(OLU_APT_SOURCES_FILE, 'a') as f:
+            f.write('deb [trusted=yes] https://debian.oresat.org/packages ./') 
+            f.close() 
+
+        # copying the apt repo signatures
+        system(f'cp {SYSTEM_SIGNATURES_DIR}* {OLU_SIGNATURES_DIR}')
```
Comment (ryanpdx) [inline update_maker/update_maker.py:69]: Don't use system(), as it is consider a bad practice.

Maybe use  listdir() and call copyfile() in a loop.

**Review by ryanpdx — COMMENTED** [review]
Other than system(), it looks good.

File: update_maker/update_maker.py (hunk 1 above)
Comment (mairacanal) [reply update_maker/update_maker.py:69]: Ok. I'll change it and make another commit.

Comment (ryanpdx) [reply update_maker/update_maker.py:69]: Also, now that I'm think about it, might want all of this to happen before cache is initialize and add a check `source.list` file exists. Therefor update_maker doesn't override all the files each time the program is started.

Comment (mairacanal) [reply update_maker/update_maker.py:69]: Okay. Probably it all will be ready by Monday.

File: update_maker/update_maker.py [hunk 2]
```diff
@@ -39,6 +44,22 @@ def __init__(self, board: str):
...
         self._cache.open()
 
+        # copying the context of the real root apt source.list file into the local one
+        if stat(OLU_APT_SOURCES_FILE).st_size == 0 or not isfile(OLU_APT_SOURCES_FILE):
+            copyfile(SYSTEM_APT_SOURCES_FILE, OLU_APT_SOURCES_FILE)    
+        
+            # adding OreSat Debian apt repo
+            with open(OLU_APT_SOURCES_FILE, 'a') as f:
+                f.write('deb [trusted=yes] https://debian.oresat.org/packages ./') 
+                f.close()
```
Comment (ryanpdx) [inline update_maker/update_maker.py:54]: `f.close()` is redundant, the `with open()` will automatically handle closing the file.

**Review by ryanpdx — CHANGES_REQUESTED** [review]
Looks good!

Rebase your commits and I will approve the PR.

**Review by ryanpdx — APPROVED** [review]

File: update_maker/update_maker.py (hunk 2 above)
Comment (mairacanal) [reply update_maker/update_maker.py]: I removed `f.close()` and rebased the commits

**Pull request merged**